File: bench/cases/stack-pop-1m/main.ts
const xs: number[] = []
const n = 1000000
for (let i = 0; i < n; i = i + 1) xs.push(i)
let total = 0
while (xs.length > 0) {
  total = total + xs.pop()!
}
console.log(total)

File: bench/cases/stack-pop-1m/main.rs
fn main() {
    let mut xs: Vec<i64> = Vec::new();
    let n: i64 = 1_000_000;
    for i in 0..n {
        xs.push(i);
    }
    let mut total: i64 = 0;
    while let Some(v) = xs.pop() {
        total += v;
    }
    println!("{}", total);
}
